src/src/NotifierStructs/N001_data_struct.cpp

```diff
@@ -25,30 +25,38 @@ bool NPlugin_001_send(const NotificationSettingsStruct& notificationsettings, St
   bool myStatus = false;
   bool failFlag = false;
 
+  WiFiClient *client = nullptr;
+
 # if FEATURE_EMAIL_TLS
 
   // values are based on the NPLUGIN_001_PKT_SZ
-  BearSSL::WiFiClientSecure_light client(4096, 4096);
-  client.setUtcTime_fcn(getUnixTime);
-  client.setCfgTime_fcn(get_build_unixtime);
-  client.setTrustAnchor(Tasmota_TA, Tasmota_TA_size);
 
-  client.setInsecure();
+  BearSSL::WiFiClientSecure_light secureClient(4096, 4096);

```

Comment:
You could make this into a pointer (set to `nullptr`) and only create one when port != 25.

Don't forget to delete it when you're done :)